content/events/2025-06-08.md

```diff
@@ -0,0 +1,22 @@
++++
+template = "event.html"
+title = "North County NUG #4"
+description = "Come join us for the 3rd North County NUG meetup!"
```

Comment:
4th*

Reply:
I'll fix that in main, no PR lol.

Reply:
Oh, repo rules force a PR :upside_down_face: